docs/CNVs/CNV-methods.tex

```diff
@@ -1010,6 +1029,50 @@ \subsubsection{Results}
 \end{figure}
 
 
+\subsection{Inferring sex genotype from raw read counts}
+The probabilistic target coverage model requires the germline copy number of the targets as an input for dealing with cohorts composed of samples with different sexes. If the phenotypic sex of the human subject associated to a sample is known, the nominal germline ploidy of each target is readily known. Unfortunately, the gender annotation is not available for most samples. In this section, we propose a simple statistical test for inferring the sex associated to a sample from human subjects from raw read counts on X and Y chromosomes.\\
```

Comment:
I would delete "If the phenotypic sex. . . not available for most samples" and change the last sentence of this paragraph to "In this section we propose a simple statistical test for inferring sex from raw read counts when the phenotypic sex is not known."